diff --git a/node_modules/@uniswap/smart-order-router/node_modules/@uniswap/v2-sdk/dist/v2-sdk.esm.js b/node_modules/@uniswap/smart-order-router/node_modules/@uniswap/v2-sdk/dist/v2-sdk.esm.js
index d977054..612cef3 100644
--- a/node_modules/@uniswap/smart-order-router/node_modules/@uniswap/v2-sdk/dist/v2-sdk.esm.js
+++ b/node_modules/@uniswap/smart-order-router/node_modules/@uniswap/v2-sdk/dist/v2-sdk.esm.js
@@ -23,7 +23,9 @@ var FACTORY_ADDRESS_MAP = {
   // BNB
   56: '0x8909Dc15e40173Ff4699343b6eB8132c65e18eC6',
   // Polygon
-  137: '0x9e5A52f57b3038F1B8EeE45F28b3C1967e22799C'
+  137: '0x9e5A52f57b3038F1B8EeE45F28b3C1967e22799C',
+  // Sepolia (testnet)
+  11155111: '0x16610120908152908b01518a1661014090815292'
 };
 var INIT_CODE_HASH = '0x96e8ac4277198ff8b6f785478aa9a39f403cb768dd02cbee326c3e7da348845f';
 var MINIMUM_LIQUIDITY = /*#__PURE__*/JSBI.BigInt(1000); // exports for internal consumption
